### ISSUE TITLE
Patched cursor.execute return value inconsistent with actual return value

### ISSUE THREAD

**Issue by Sazpaimon** [DataDog/dd-trace-py#799]
When upgrading to ddtrace 0.20 from 0.16, I discovered that cursor.execute returns another cursor. In MySQLdb, the return value should be number of rows. I'm running Python 2.7.15 on Ubuntu 18.04
Simple test case:

Without patching:
```
import MySQLdb
db = MySQLdb.connect(user='root', host='mysql')
print(db.cursor().execute('SELECT 1'))
1
```

With patching:
```
from ddtrace import patch_all
patch_all()

import MySQLdb
db = MySQLdb.connect(user='root', host='mysql')
print(db.cursor().execute('SELECT 1'))
<MySQLdb.cursors.Cursor object at 0x7fbbc4b23810>
```

**Comment by majorgreys** [comment]
@Sazpaimon I want to confirm that we are able to reproduce this. We will look more closely at this issue and loop back with you. Thank you for pointing this behavior out!

**Comment by goodspark** [comment]
I'm seeing this as well.

**Comment by brettlangdon** [comment]
Fixed by #830 

This fix should be released next week as part of the `0.21.1` release.

**Comment by brettlangdon** [comment]
`0.21.1` has been released which fixes this issue.
